app/src/components/CustomPeriodDialog.tsx

```diff
@@ -25,16 +25,20 @@ export function CustomPeriodDialog(props: CustomPeriodDialogProps) {
   const [endTime, setEndTime] = useState<TimeValue>(new Time(12, 0));
 
   const [startDate, setStartDate] = useState<DateValue>(
-    toCalendarDate(new Date(Date.now() - PeriodProps[Period.WEEK].milliseconds))
+      toCalendarDate(new Date(
+          Math.max(Date.now() -
+              PeriodProps[Period.WEEK].milliseconds,
+              new Date('2013-01-01T00:00:00').getTime())))
   );
 
   const [endDate, setEndDate] = useState<DateValue>(toCalendarDate(new Date()));
 
   function handleSelection() {
-    const startDateTime = toDate(startDate, startTime);
+    const startDateTime = Math.max(toDate(startDate, startTime).getTime(),
+        new Date('2013-01-01T00:00:00').getTime());
```

Comment:
Also nitpick, but could we set this to being at mid day instead of midnight? Because otherwise it might be converted to december 31st 2012 in some timezones.

Or maybe even February 22th (OSRS launch day), just to avoid any chance at converting to 2012 🤷

Reply:
```javascript
/* minDate = OSRS Launch Date Febuary 22nd 2012 @ 12:00:00 PM */
  const minDate = new Date('2012-02-22T12:00:00');
  const [startDate, setStartDate] = useState<DateValue>(
      toCalendarDate(new Date(
          Math.max(new Date(minDate).getTime())))
  );
```

Reply:
That snippet is incomplete, but yea that's the gist